supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -1008,16 +1011,18 @@ Use the name of the tab when you refer to it, for example, "the *Storage* tab".
 *See also*:
 
 // Fuse: Added "In Red Hat Fuse,"
+// Fuse: Removed two sentences about Fuse tooling (Breda)
+// Fuse: Removed "Source tab" from "See also" (Breda)
 [discrete]
 [[routing-rules]]
 ==== image:images/yes.png[yes] routing rules (noun)
-*Description*: In Red Hat Fuse, routing rules are declarative statements (written in Java or XML DSL) that define the paths which messages take from their origin (source) to their target destination (sink). Routing rules start with a consumer endpoint (`from`) and typically end with one or more producer endpoints (`to`). Between consumer and producer endpoints, messages can enter various processors, which might transform them or redirect them to other processors or to specific producer endpoints. In Fuse tooling, you can view and edit a project's routing rules on the route editor's Source tab. On the Design tab, you can build and view routing rules graphically.
+*Description*: In Red Hat Fuse, routing rules are declarative statements (written in Java or XML DSL) that define the paths which messages take from their origin (source) to their target destination (sink). Routing rules start with a consumer endpoint (`from`) and typically end with one or more producer endpoints (`to`). Between consumer and producer endpoints, messages can enter various processors, which might transform them or redirect them to other processors or to specific producer endpoints.
```

Comment:
IBM SG tends to reserve parentheses for unique cases, such as abbreviations. Consider:

```suggestion
*Description*: In Red Hat Fuse, routing rules are declarative statements, which are written in Java or XML DSL, that define the paths which messages must take from their origin, or source, to their target destination, which is also known as a _sink_. Routing rules start with a consumer endpoint, `from`, and typically end with one or more producer endpoints, `to`,. Between consumer and producer endpoints, messages can enter various processors, which might transform them or redirect them to other processors or to specific producer endpoints.
```

Reply:
I take your point and will rewrite in a way that I hope satisfies us both :)